src/GestureManager.cpp

```diff
@@ -123,11 +124,27 @@ bool GestureManager::handleDragGesture(const DragGestureEvent& gev) {
     auto const workspace_swipe_edge_str = std::string{*WORKSPACE_SWIPE_EDGE};
 
     switch (gev.type) {
-        case DragGestureType::SWIPE:
-            if (**WORKSPACE_SWIPE_FINGERS != gev.finger_count) {
-                return false;
+        case DragGestureType::SWIPE: {
+            if (**WORKSPACE_SWIPE_FINGERS == gev.finger_count) {
+                if (!g_pGestureManager->handleWorkspaceSwipe(gev.direction)) {
+                    static auto* const PEVENTVEC = g_pHookSystem->getVecForEvent("hyprgrass:swipeStart");
+                    SCallbackInfo info;
+
+                    g_pHookSystem->emit(PEVENTVEC, info,
+                                        IPointer::SSwipeBeginEvent{.fingers = (uint32_t)gev.finger_count});
+
+                    if (!info.cancelled) {
```

Comment:
@horriblename I figured out how to add it back, but to make it work with hyprexpo I had to check if `info.cancelled` was false instead of true

Reply:
@horriblename Is this expected? I thought it would've been the opposite no?